src/test/integration/mocks/BeaconChainMock.t.sol

```diff
@@ -191,17 +198,21 @@ contract BeaconChainMock is Test {
         proof.validatorFields[0][BeaconChainProofs.VALIDATOR_BALANCE_INDEX] = 
             _toLittleEndianUint64(validator.effectiveBalanceGwei);
 
+        emit log("10");
```

Comment:
oh man, whatever led you to having to do this, i feel for you haha

Reply:
is it fixed now? we should remove if so

Reply:
fixed now, much pain